src/shared/utils/get-standardised-invoice-key.test.ts

```diff
@@ -5,7 +5,8 @@ test("Invoice key getter with standardised line item", () => {
     invoice_receipt_id: "370 000",
     vendor_id: "vendor_testvendor1",
     vendor_name: "Vendor One",
-    invoice_receipt_date: "2023-02-28",
+    invoice_receipt_date: "2023-03-01",
```

Comment:
is there a reason to keep the "invoice_receipt_date" (in all instances), as we are no longer using it?

Reply:
Didn't want to risk breaking things on a Friday afternoon, and it can't hurt to carry it along, so I left it in.  Even if we aren't now using it, we might want to show this value in the future.